config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanics/:id", to: "mechanics#show"
+  get "/amusement_parks/:id", to: "amusement_parks#show"
+
+  post "/ride_mechanics/", to: "ride_mechanics#create"
```

Comment:
If you used the alternative `url` that I mentioned your route would then look like
```ruby
post "/mechanics/:id/rides", to: "ride_mechanics#create"
```
This looks good.